packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -0,0 +1,271 @@
+
+  const response = await sendRequestToRelay(request);
+
+  return response.result;
+}
+
+async function getLatestBlockHash() {
+  const request = {
+    jsonrpc: '2.0',
+    method: 'eth_getBlockByNumber',
+    params: ['latest', false],
+    id: 0,
+  };
+
+  const response = await sendRequestToRelay(request);
+
+  return response.result.hash;
+}
+
+function splitReqAndRes(content) {
+  const lines = content.split('\n');
+  const filteredLines = lines.filter((line) => line != '').map((line) => line.slice(3));
+
+  return { request: filteredLines[0], response: filteredLines[1] };
+}
+
+async function sendRequestToRelay(request) {
+  try {
+    const response = await axios.post(relayUrl, request);
+    return response.data;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+}
+
+async function signAndSendRawTransaction(transaction) {
+  const transactionCount = await getTransactionCount();
+  transaction.nonce = parseInt(transactionCount, 16);
+  const signed = await signTransaction(
+    transaction,
+    '0x6e9d61a325be3f6675cf8b7676c70e4a004d2308e3e182370a41f5653d52c6bd',
+  );
+  const request = {
+    jsonrpc: '2.0',
+    id: 1,
+    method: 'eth_sendRawTransaction',
+    params: [signed],
+  };
+
+  const response = await sendRequestToRelay(request);
+  const requestTransactionReceipt = {
+    id: 'test_id',
+    jsonrpc: '2.0',
+    method: 'eth_getTransactionReceipt',
+    params: [response.result],
+  };
+  const transactionReceipt = await sendRequestToRelay(requestTransactionReceipt);
+  return {
+    transactionHash: response.result,
+    blockHash: transactionReceipt.result.blockHash,
+    transactionIndex: transactionReceipt.result.transactionIndex,
+    blockNumber: transactionReceipt.result.blockNumber,
+  };
+}
+
+async function checkRequestBody(fileName, request) {
+  //This method changes params for request which wont work with the values from ethereum api tests
+  if (
+    (request.method === 'eth_getBlockByHash' &&
+      request.params[0] === '0x7cb4dd3daba1f739d0c1ec7d998b4a2f6fd83019116455afa54ca4f49dfa0ad4') ||
+    (request.method === 'eth_sendRawTransaction' &&
+      request.params[0] ===
+        '0xf86709843b9aca018261a894aa000000000000000000000000000000000000000a825544820a95a0281582922adf6475f5b2241f0a4f886dafa947ecdc5913703b7840344a566b45a05f685fc099161126637a12308f278a8cd162788a6c6d5aee4d425cde261ba35d')
+  ) {
+    request.params[0] = currentBlockHash;
+  }
+  if (request.method === 'eth_getTransactionByBlockHashAndIndex') {
+    request.params[0] = legacyTransactionAndBlockHash.blockHash;
+    request.params[1] = legacyTransactionAndBlockHash.transactionIndex;
+  }
+  if (request.method === 'eth_getTransactionByBlockNumberAndIndex') {
+    request.params[0] = legacyTransactionAndBlockHash.blockNumber;
+    request.params[1] = legacyTransactionAndBlockHash.transactionIndex;
+  }
+  if (request.method === 'eth_sendRawTransaction') {
+    const transactionCount = await getTransactionCount();
```

Comment:
There are some redundant variables like this one.

Reply:
eslint does not show me this error, so could you please state all the redundant variables that your ide warns you for?